### PR TITLE
Update docs for Jan 2024 Connectathon

### PR DESCRIPTION
**Pull request by jtquach1**
## Describe your changes

> Please include a summary of the changes and the related issue/task. Please also include relevant motivation and context. List any dependencies that are required for this change, including links to other pull requests/branches in other repositories if applicable.

Alphabetized env vars, updated README, updated NOTICE copyright year, and deleted unused files.

## Issue ticket number and Jira link

> Please include the Jira Ticket Number and Link for this issue/task.

https://jira.mitre.org/browse/REMS-556

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,91 @@
...
-    To stop the running container, simply use ctrl + c
 
+The .env file contains the default URI paths, which can be overwritten from the start command as follows:
+a) `REACT_APP_LAUNCH_URL=http://example.com PORT=6000 npm start` or b) by specifying the environment variables and desired values in a `.env.local`.
+
+> **Bug**: Do note that the `SMART_ENDPOINT` environment variable cannot be overwritten in a `.env.local`, it must be done in the `.env`.
+
+Following are a list of modifiable paths:
+
+| URI Name        | Default                                    |
```
Comment (KeeyanGhoreshi) [inline README.md:23]: not necessarily a change we need right now, but this list could use some sort of description of what each variable is for and why you'd change it.

File: patches/@projecttacoma+node-fhir-server-core+2.2.8.patch [hunk 2]
```diff
@@ -1,16 +0,0 @@
-diff --git a/node_modules/@projecttacoma/node-fhir-server-core/dist/server/resources/4_0_0/schemas/datarequirement.js b/node_modules/@projecttacoma/node-fhir-server-core/dist/server/resources/4_0_0/schemas/datarequirement.js
```
Comment (KeeyanGhoreshi) [inline patches/@projecttacoma+node-fhir-server-core+2.2.8.patch:1]: why was this patch deleted? Has the bug with datarequirment been fixed in the package?

Comment (jtquach1) [reply patches/@projecttacoma+node-fhir-server-core+2.2.8.patch:1]: oh interesting, I didn't know this was useable code

Comment (jtquach1) [reply patches/@projecttacoma+node-fhir-server-core+2.2.8.patch]: sorry about that, I misunderstood what the patch meant- just cherry-picked the commit and removed the removal of the patch and force pushed

File: README.md (hunk 1 above)
Comment (jtquach1) [reply README.md:23]: should we make a ticket for this (esp if it might be useful for the connectathon)?

**Review by KeeyanGhoreshi — APPROVED** [review]
just needs conflict fixes

File: README.md (hunk 1 above)
Comment (plarocque4) [reply README.md:23]: REMS-590 was created for this.

**Pull request merged**